composer.json

```
@@ -29,8 +29,9 @@
     "oat-sa/generis": ">=14.2.0",
     "oat-sa/tao-core": ">=48.0.0",
     "oat-sa/extension-tao-delivery": ">=14.10.1",
-    "oat-sa/extension-tao-outcomeui": ">=10.4.0"
-  },
+    "oat-sa/extension-tao-outcomeui": ">=10.4.0",
+    "oat-sa/lib-tao-elasticsearch": "1.2.0"
```

If we are going to include it, should it be a bit less strict version restriction 
```suggestion
    "oat-sa/lib-tao-elasticsearch": "^1.2.0"
```

---

> But should we? I've thought idea was we may have different implementations in different libs, connected with help of this extensions to the TAO. So I agree that `oat-sa/lib-tao-elasticsearch` may go under `suggestion` settings of composer, but isn't too much to put under hard requirements?

But we have dependencies from this library in this extension. We cannot install it without library.